configs/meg4B.yml

```diff
@@ -0,0 +1,88 @@
+# GPT-2 pretraining setup
```

Comment:
could you get rid of all these benchmarking configs from the PR? They weren't meant to be merged into the main branch